mqml/tests/test_conductresist.py

```diff
@@ -0,0 +1,78 @@
+import pytest
+
+from mqml.parameter.conductresist import ConductResist
+from qcodes.instrument.base import Instrument
+import numpy as np
+
+@pytest.fixture(autouse=True)
+def close_all_instruments():
+    """Makes sure that after startup and teardown all instruments are closed"""
+    Instrument.close_all()
+    yield
+    Instrument.close_all()
+
+def test_get_static_parameters():
+    "This tests the initial values of static parameters in the class"
+    lockin_param1 = 1
+    lockin_param2 = 2
+    test_param = ConductResist('test_param', lockin_param1=lockin_param1, lockin_param2=lockin_param2)
+
+    assert test_param.GIamp() == None
+    assert test_param.GVamp() == None
+    assert test_param.ACdiv() == 1e-4
+    assert test_param.DCdiv() == 1e-2
+
+def test_errors():
+    "This tests the erros"
+    lockin_param1 = 1
+    lockin_param2 = 2
+    test_param = ConductResist('test_param', lockin_param1=lockin_param1, lockin_param2=lockin_param2)
+
+    with pytest.raises(TypeError) as e:
+        test_param.diff_conductance_fpm()
+    assert str(e.value) == "(\'Amplification and/or voltage divisions are not set. Set them and try again.\', \'getting "\
+                            "test_param_diff_conductance_fpm\')"
+
+    with pytest.raises(TypeError) as e:
+        test_param.conductance_tpm()
+    assert str(e.value) == "(\'Amplification and/or voltage divisions are not set. Set them and try again.\', \'getting "\
+                            "test_param_conductance_tpm\')"
+
+    with pytest.raises(TypeError) as e:
+        test_param.resistance_fpm()
+    assert str(e.value) == "(\'Amplification and/or voltage divisions are not set. Set them and try again.\', \'getting "\
+                            "test_param_resistance_fpm\')"
+
+def test_warnings():
+    "This tests warnings if zero divisions occur"
+    lockin_param1 = 1.0
+    lockin_param2 = 0.0
+    test_param = ConductResist('test_param', lockin_param1=lockin_param1, lockin_param2=lockin_param2)
+    test_param.GIamp(1e7)
+    test_param.GVamp(100.0)
+    nan = np.nan
+
```

Comment:
Would be nice to use the pytest warning context manager here to assert that this raises a warning. https://docs.pytest.org/en/3.0.5/recwarn.html